### PR TITLE
Update Language.nl.xml

### PR DESCRIPTION
**Pull request by RockyTDR**
Very small language update

### CODE REVIEW

File: Translations/Language.nl.xml [hunk 1]
```diff
@@ -1455,7 +1455,7 @@ Wisselbestanden kunnen zich alleen op de systeempartitie bevinden.</entry>
     <entry lang="nl" key="IDC_FORCE_VERACRYPT_FIRST_BOOT_ENTRY">Zorgen dat VeraCrypt als eerste wordt weergegeven in het EFI-firmware-opstartmenu.</entry>
     <entry lang="nl" key="RAM_ENCRYPTION_DISABLE_HIBERNATE">WAARSCHUWING: RAM-versleuteling is niet compatibel met Windows-functies sluimerstand en "snel opstarten". VeraCrypt moet deze uitschakelen voordat de RAM-versleuteling wordt geactiveerd.\n\nDoorgaan?</entry>
     <entry lang="nl" key="CONFIRM_DISABLE_FAST_STARTUP">WAARSCHUWING: Windows snel opstarten is ingeschakeld en het is bekend dat het problemen veroorzaakt bij het werken met VeraCrypt-volumes. Het is raadzaam om het uit te schakelen voor een betere beveiliging en gebruiksvriendelijkheid.\n\nWilt u Windows snel opstarten uitschakelen?</entry>
- </localization>
+  </localization>
```
Comment (alt3r-3go) [inline Translations/Language.nl.xml:1458]: This is an unnecessary whitespace change, which essentially just pollutes git history - please consider removing it.

**Review by alt3r-3go — CHANGES_REQUESTED** [review]

File: Translations/Language.nl.xml (hunk 1 above)
Comment (RockyTDR) [reply Translations/Language.nl.xml]: @alt3r-3go Done! (I think...)
